.github/actions/use-pre-commit/action.yml

```diff
@@ -30,29 +30,21 @@ runs:
         key: pre-commit-${{ inputs.version }}-${{ hashFiles(inputs.config-file) }}
         path: |
           ~/.cache/pre-commit
-          ~/.pre-commit-venv
-
-    - name: Create virtualenv
-      if: steps.cache-pre-commit.outputs.cache-hit != 'true'
-      run: python -m venv ~/.pre-commit-venv
-      shell: bash
 
     - name: Install pre-commit
       if: steps.cache-pre-commit.outputs.cache-hit != 'true'
       run: |
-        source ~/.pre-commit-venv/bin/activate
-        python -m pip install pre-commit==2.19.0
+        # Install pre-commit as a standalone .pyz archive that can be run by Python
```

Comment:
Move this comment in the `name` field ? It feel appropriate to be placed here

Reply:
ok

Reply:
I was expecting to be put in the field `name`, to produce something like

```yaml
name: Install pre-commit as a standalone .pyz archive that can be run by Python
```